src/color-inline/README.md

```diff
@@ -28,4 +34,38 @@ Invalid color:
 
 ```html
 <color-inline>foobar</color-inline>
-```
\ No newline at end of file
+```
+
+## Reference
+
+### Slots
+
+| Name | Description |
+|------|-------------|
+| (default) | The element's main content—the color to be shown. Placed next to the color swatch. |
+
+### Attributes & Properties
+
+| Attribute | Property | Property type | Default value | Description |
+|-----------|----------|---------------|---------------|-------------|
+| `color` | `color` | `Color` &#124; `null` | - | The current color value. `null` for invalid colors. |
+| `value` | `value` | `string` &#124; `undefined` | - | The current value. `undefined` if the `value` attribute is not set. |
+
+
+### CSS variables
+
+| Variable | Type | Description |
+|----------|---------------|-------------|
+| `--transparency-grid` | `<image>` | Gradient used as a background for transparent parts of the swatch. |
+| `--transparency-tile-size` | `<length>` | The size of the tiles in the transparency grid. |
```

Comment:
```suggestion
| `--transparency-tile-size` | `<length>` | The size of the tiles in the transparency grid. This will not be used if you are overriding `--transparency-grid` |
```

Reply:
Also, can we make sure these are harmonized across all components that have a transparency grid?